spec/features/stats_spec.rb

```diff
@@ -1,12 +1,41 @@
-RSpec.feature 'View usage stats', :vcr do
+RSpec.describe 'View usage stats', :vcr, type: :feature do
+  include Warden::Test::Helpers
+
   let(:user) { create(:user, roles: ['data_analyst']) }
 
   before do
-    sign_in user
-    UnlinkReason.create! code: 1, description: "Linked to wrong case ID (correct defendant)"
-    UnlinkReason.create! code: 7, description: "Other"
+    Warden.test_mode!
+    login_as(user, scope: :user)
+
+    # Create required data
+    UnlinkReason.create!(code: 1, description: "Linked to wrong case ID (correct defendant)")
+    UnlinkReason.create!(code: 7, description: "Other")
+
+    # Stub external calls for predictable data
```

Comment:
It's okay for now,
but it's not ideal!

This is not a unit test, we should stub at API level.
In the next ticket, let's use the the VCR/cassettes or Webmock (https://github.com/bblimke/webmock)

Please add a TODO comment to support VCR/cassettes or Webmock